Makefile

```diff
@@ -190,7 +190,10 @@ install.tools: .install.gitvalidation .install.golangci-lint .install.vndr ## in
 
 .install.golangci-lint:
 	@echo "$(WHALE) $@"
-	$(GO) get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+	$(GO) get -d github.com/golangci/golangci-lint/cmd/golangci-lint
+	@cd $(GOPATH)/src/github.com/golangci/golangci-lint/cmd/golangci-lint; \
```

Comment:
suggestion... let's open an issue (or add a line to an existing one) to use containerd/script/setup/install-dev-tools, maybe also move said script to containerd/project

Reply:
Done https://github.com/containerd/project/issues/35